build.sbt

```diff
@@ -3,34 +3,35 @@ import SonatypeKeys._
 val commonSettings = Vector(
   name := "shocon",
   organization := "org.akka-js",
-  version := "0.2.1",
-  scalaVersion := "2.12.2",
-  crossScalaVersions  :=
-    Vector("2.11.11", "2.12.2")
+  version := "0.3.0-SNAPSHOT",
+  scalaVersion := "2.12.4",
```

Comment:
base version should be 2.11, easier to run all tests by default
* since only 2.11 is cross-3 platform (JVM JS Native)
* and 2.12 is only cross-2 platform (JVM JS)

Reply:
sounds ok...